runbot_docker_orchest/tests/test_runbot_build.py

```diff
@@ -0,0 +1,119 @@
+from odoo.tests.common import TransactionCase
+from odoo.tools.misc import mute_logger
+import xmlrpc.client
+import logging
+import time
+import unittest
+import os
+import subprocess
+from json import load
+
+_logger = logging.getLogger(__name__)
+
+
+def docker_login(filename):
+    filename = os.path.expanduser(filename)
+    is_login = False
+    if not os.path.exists(filename):
+        return False
+    try:
+        is_login = load(
+            open(filename)).get('auths', {}).get('quay.io') or False
```

Comment:
Locally the tests are not running...
Because my configuration file is:
```json
{
  "auths" : {
    "https://index.docker.io/v1/" : {
    },
    "quay.io" : {
    }
  },
```

I just apply the following patch
```diff
index 05e79ae..ee624ec 100644
--- a/runbot_docker_orchest/tests/test_runbot_build.py
+++ b/runbot_docker_orchest/tests/test_runbot_build.py
@@ -13,14 +13,10 @@ _logger = logging.getLogger(__name__)

 def docker_login(filename):
     filename = os.path.expanduser(filename)
-    is_login = False
-    if not os.path.exists(filename):
-        return False
     try:
-        is_login = load(
-            open(filename)).get('auths', {}).get('quay.io') or False
-    except ValueError:
-        return False
+        is_login = 'quay.io' in load(open(filename)).get('auths')
+    except (ValueError, TypeError, FileNotFoundError):
+        is_login = False
     return is_login

```